addons/test_mail_full/tests/test_rating.py

```diff
@@ -148,3 +150,14 @@ def test_rating_last_value_perfs(self):
                 record_ratings._compute_rating_last_value()
                 vals = [val == 5 for val in record_ratings.mapped('rating_last_value')]
                 self.assertTrue(all(vals), "The last rating is kept.")
+
+
+class TestRatingRoutes(HttpCase, TestRatingFlow):
```

Comment:
Actually, would be better to have ``TestRatingFlowCommon`` with the setup, then ``TestRatingFlow`` and ``TestRatingRoutes``. Otherwise when running TestRatingRoutes, it runs all tests from TestRatingFlow again (hence twice the error in runbot)